packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -288,21 +296,23 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
         bytes32 transactionId = keccak256(request);
         BridgeTransactionV2 memory transaction = getBridgeTransactionV2(request);
 
+        BridgeTxDetails storage $ = bridgeTxDetails[transactionId];
```

Comment:
Similarly here, if we were to do all the `$` reads at this point into the stack variables, we would see some extra gas savings on top.